### ISSUE TITLE
Maybe it is a bug in mt_dnn/model.py

### ISSUE THREAD

**Issue by leewea** [namisan/mt-dnn#196]
![bug](https://user-images.githubusercontent.com/20851491/101165943-701c3300-3672-11eb-9cbc-9af49ec9f494.png)
![bug显示](https://user-images.githubusercontent.com/20851491/101166062-a2c62b80-3672-11eb-8e2c-ff0d02560064.png)
modify to loss.item() can work 
`self.train_loss.update(loss.item(), batch_size) `

**Comment by nanzhao** [comment]
should be like this ?      
      if self.config['local_rank'] != -1:
            # print('Rank ', self.config['local_rank'], ' loss ', loss)
            copied_loss = copy.deepcopy(loss.data)
            torch.distributed.all_reduce(copied_loss)
            copied_loss = copied_loss / self.config['world_size']
            self.train_loss.update(copied_loss.item(), batch_size)
        else:
            self.train_loss.update(loss.item(), batch_size)

**Comment by namisan** [comment]
thanks @nanzhao , I've already fixed the bug. I recently imp-ed dynamic bins and DDP to speed up training, and it works quite well. Will update instructions.
